packages/npm-dependencies-validation/test/utils/npmUtil.spec.ts

```diff
@@ -1,15 +1,55 @@
+import { fail } from "assert";
```

Comment:
using `fail` from `assert` is strange as `chai` provides it's own `assert.fail`
But I think we may not need `fail` at all

Reply:
Done